org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/ConfirmPanel.java

```diff
@@ -259,8 +259,15 @@ private void init()
 
         // IDEMPIERE-1334 start
         pnlBtnCenter = new Hlayout();
-        pnlBtnCenter.setSclass("confirm-panel-center");
+        pnlBtnCenter.setSclass("z-valign-middle confirm-panel-center");
+        pnlBtnCenter.setStyle("height: 100%");
```

Comment:
setHeight("100%") is prefer over setstyle here.

Reply:
done